### ISSUE TITLE
Incorrect positions on disambiguation result of pdf files

### ISSUE THREAD

**Issue by tantikristanti** [kermitt2/nerd#67]
After having some tests on disambiguation for Pdf files for all languages (English, French, German, Italian, and Spanish), it was found some incorrect positions of coordinate (X,Y) and running pointers for mentions in the Title part.

<img width="987" alt="french local title" src="https://user-images.githubusercontent.com/28044697/36385636-51b3504a-1593-11e8-8a88-614006ddef43.png">

Furthermore, for the French Pdf file, it was also found some mentions that didn't point to the correct entities. 

<img width="1095" alt="french local content" src="https://user-images.githubusercontent.com/28044697/36385783-dc6e0806-1593-11e8-994c-df8a23b4b418.png">

The source of Pdf file used for the French disambiguation service can be found here [French](https://github.com/kermitt2/nerd/blob/0.0.3/data/corpus/corpus-long/hirmeos/originals-pdfs/ahrf-9992.pdf)

**Comment by lfoppiano** [comment]
Godfroy has 4 occurrence, 3 in page 5 and 1 in page 18. 

Nerd find it 8 times: 

<img width="1064" alt="screen shot 2018-02-26 at 14 06 52" src="https://user-images.githubusercontent.com/15426/36672102-5cc52d6c-1afe-11e8-831b-c970fdf8bb06.png">


From the json output we get some frenzy entities: 

```
    {
      "rawName": "Godfroy",
      "offsetStart": 12138,
      "offsetEnd": 12145,
      "pos": [
        {
          "p": 5,
          "x": 226.224,
          "y": 411.066,
          "w": 30.893099999999997,
          "h": 10.9257
        }
      ],
      "nerd_score": 0,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    },
[...]
{
      "rawName": "Godfroy",
      "offsetStart": 12368,
      "offsetEnd": 12375,
      "pos": [
        {
          "p": 5,
          "x": 85.0391,
          "y": 453.312,
          "w": 32.9972,
          "h": 10.9257
        }
      ],
      "nerd_score": 0,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    }, 
[...]
{
      "rawName": "Godfroy",
      "offsetStart": 12772,
      "offsetEnd": 12779,
      "pos": [
        {
          "p": 5,
          "x": 221.956,
          "y": 509.675,
          "w": 33.007,
          "h": 10.9257
        }
      ],
      "nerd_score": 0,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    },
[...]
 {
      "rawName": "Godfroy",
      "type": "PERSON",
      "offsetStart": 13763,
      "offsetEnd": 13770,
      "pos": [
        {
          "p": 5,
          "x": 436.59,
          "y": 668.179,
          "w": 16.5434,
          "h": 10.9257
        },
        {
          "p": 5,
          "x": 85.0391,
          "y": 682.261,
          "w": 4.301072727272727,
          "h": 10.9257
        }
      ],
      "nerd_score": 0.8,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    },
[...]
{
      "rawName": "Godfroy",
      "type": "PERSON",
      "offsetStart": 13993,
      "offsetEnd": 14000,
      "nerd_score": 0.8,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    },
[...]
{
      "rawName": "Godfroy",
      "type": "PERSON",
      "offsetStart": 14397,
      "offsetEnd": 14404,
      "pos": [
        {
          "p": 6,
          "x": 312.22,
          "y": 72.8153,
          "w": 7.6310399999999845,
          "h": 10.925700000000006
        }
      ],
      "nerd_score": 0.8,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    },
[...]
 {
      "rawName": "Godfroy",
      "offsetStart": 36954,
      "offsetEnd": 36961,
      "pos": [
        {
          "p": 18,
          "x": 159.921,
          "y": 383.316,
          "w": 32.9972,
          "h": 10.9257
        }
      ],
      "nerd_score": 0,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    },
[...]
{
      "rawName": "Godfroy",
      "type": "PERSON",
      "offsetStart": 67846,
      "offsetEnd": 67853,
      "pos": [
        {
          "p": 18,
          "x": 159.921,
          "y": 383.316,
          "w": 32.9972,
          "h": 10.9257
        }
      ],
      "nerd_score": 0.8,
      "nerd_selection_score": 0.4301,
      "wikipediaExternalRef": 1657375,
      "wikidataId": "Q2568594",
      "domains": [
        "Military"
      ]
    },
```

**Comment by lfoppiano** [comment]
This issue should have been solved in commit a985a7706fca656748a6c113a27054d91729de0c

**Comment by lfoppiano** [comment]
To be checked: 

<img width="491" alt="screen shot 2018-03-01 at 16 00 30" src="https://user-images.githubusercontent.com/15426/36851545-bd9d4aee-1d69-11e8-871d-0330b9ca64fc.png">

in this file: 
[Shark_Attacks_on_Bottlenose_Dolphins_Tur.pdf](https://github.com/kermitt2/nerd/files/1771597/Shark_Attacks_on_Bottlenose_Dolphins_Tur.pdf)
